docs: add protoc install steps (#1038)
tendermint/starport: docs/intro/install.md

sudo mv starport /usr/local/bin/
```

### Error while running command `protoc`

> If you get errors related to `protoc`, please, make sure that you followed [the instructions](https://grpc.io/docs/protoc-installation/#install-pre-compiled-binaries-any-os):

Reviewer: ```suggestion
If you get errors related to `protoc`, make sure that you followed the [Install pre-compiled binaries (any OS)](https://grpc.io/docs/protoc-installation/#install-pre-compiled-binaries-any-os) instructions:
```

> 1. Manually download from github.com/google/protobuf/releases the zip file corresponding to your operating system and computer architecture (`protoc-<version>-<os><arch>.zip`).

Reviewer: ```suggestion
1. From [github.com/google/protobuf/releases](github.com/google/protobuf/releases), manually download the zip file that corresponds to your operating system and computer architecture (`protoc-<version>-<os><arch>.zip`).
```

Author: Updated the commend to add `https://` to the link.

> 2. Unzip the file under $HOME/.local or a directory of your choice. For example:

Reviewer: ```suggestion
2. Unzip the file under `$HOME/.local` or a directory of your choice. For example:
```

```
$ unzip protoc-3.15.5-linux-x86_64.zip -d $HOME/.local
```

> 3. Update your environment’s path variable to include the path to the protoc executable. For example:

Reviewer: ```suggestion
3. Update your environment’s PATH variable to include the path to the protoc executable. For example:
```
